packages/assistant/src/RPA/Assistant/library.py

```diff
@@ -0,0 +1,988 @@
+            Add file input    name=dest      destination=%{ROBOT_ROOT}
+
+            # This restricts files to certain types
+            Add file input    name=types     file_type=PDF files (*.pdf)
+
+            # Every file input result is a list of paths
+            ${result}=    Run dialog
+            FOR    ${path}    IN    @{result.multiple}
+                Log    Selected file: ${path}
+            END
+        """
+
+        def on_pick_result(e: FilePickerResultEvent):
+            if e.files:
+                self._client.results[str(name)] = list(map(lambda f: f.path, e.files))
+
+        file_picker = FilePicker(on_result=on_pick_result)
+        self._client.add_invisible_element(file_picker)
+
+        options = {
+            "source": optional_str(source),
+            "destination": optional_str(destination),
+            "file_type": optional_str(file_type),
+        }
+
+        if not options["source"]:
+            options["source"] = os.path.expanduser("~")
+
+        self._client.add_element(
+            ElevatedButton(
+                label or "Choose files...",
+                on_click=lambda _: file_picker.pick_files(
+                    allow_multiple=bool(multiple),
+                    initial_directory=options["destination"],
+                    allowed_extensions=options["file_type"],
+                ),
+            )
+        )
+
+    @keyword("Add drop-down", tags=["input"])
+    def add_drop_down(
+        self,
+        name: str,
+        options: Options,
+        default: Optional[str] = None,
+        label: Optional[str] = None,
+    ) -> None:
+        """Add a drop-down element
+
+        :param name:    Name of result field
+        :param options: List of drop-down options
+        :param default: The default selection
+        :param label:   Label for input field
+
+        Creates a drop-down menu with the given ``options``. The selection
+        the user makes will be available in the ``name`` field of the result.
+
+        The ``default`` argument can be one of the defined options,
+        and the dialog automatically selects that option for the input.
+
+        A custom ``label`` text can also be added.
+
+        Example:
+
+        .. code-block:: robotframework
+
+            Add heading     Select user type
+            Add drop-down
+            ...    name=user_type
+            ...    options=Admin,Maintainer,Operator
+            ...    default=Operator
+            ...    label=User type
+            ${result}=      Run dialog
+            Log    User type should be: ${result.user_type}
+
+        """
+        options, default = to_options(options, default)
+
+        options: List[Control] = list(map(Option, options))
+
+        dropdown = Dropdown(options=options, value=default)
+
+        self._client.add_element(Text(value=label))
+        self._client.add_element(dropdown, name=str(name))
+
+    # FIXME: Add keyword back when fixed
```

Comment:
Would help to provide more info on what is not working so it has to be fixed before uncommenting this.

Reply:
done, inside the class